.gitignore

```diff
@@ -20,3 +20,4 @@ bin/*
 !bin/install
 !bin/sylius-cs-fixer
 vendor
+/.idea/
```

Comment:
You can also use a global ignore file.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore